packages/cli/src/license/License.service.ts

```diff
@@ -31,6 +32,7 @@ export class LicenseService {
 				planId: mainPlan?.productId ?? '',
 				planName: license.getPlanName(),
 			},
+			environment: config.getEnv('license.tenantId') === 1 ? 'production' : 'staging',
```

Comment:
@krynble would it be too much to ask to move this under the `GET /rest/settings` response?

this could come handy in other situations as well

Reply:
![image](https://user-images.githubusercontent.com/219272/208119832-bb66cd2a-51f7-4aa5-979e-ba01dc3feeb2.png)
Sure, done @cstuncsik

Reply:
@krynble I thought `environment` prop should be just next to  `logLevel` for example and not nested under `license`

Reply:
Hey @cstuncsik I've placed it there to avoid confusion because it relates only to the license server by detecting the tenant ID only. If I placed it in the top level it could cause confusion in the future and be used for other stuff rather than just controlling redirect links to the stage app.

I can change it if needed but I chose to implement it like this to avoid confusion.

Reply:
ok, used it from ther